content/content-strategy/manage-content-requests/longtext.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: text/textblock
+---
+#### [1]Content production lifecycle: long text description
```

Comment:
Remove `: long text description` from the heading. It's now automatically shown on the accordion used to hide/show the description. Only the heading is needed.

Reply:
Trisha: done.